cmd/soroban-cli/src/commands/network/stop.rs

```diff
@@ -1,7 +1,12 @@
 use std::process::Command;
 
 #[derive(thiserror::Error, Debug)]
-pub enum Error {}
+pub enum Error {
+    #[error("Failed to execute docker command: {error}")]
```

Comment:
This is totally fine way to do it, but my usual preference is to use {0} and then the enum case can be a tuple:
```rust
CommandError(std::id::Error)
```
then the map becomes:
```rust
.map_err(Error::CommandError)
```